spikeinterface/comparison/multisortingcomparison.py

```diff
@@ -1,19 +1,19 @@
 import numpy as np
 from pathlib import Path
 import json
-import os
 
 from spikeinterface.core import load_extractor, BaseSorting, BaseSortingSegment
-from .basecomparison import BaseComparison
+from .basecomparison import BaseSpikeTrainComparison
+from .basemulticomparison import BaseMultiComparison
 from .symmetricsortingcomparison import SymmetricSortingComparison
 from .comparisontools import compare_spike_trains
 
 import networkx as nx
 
 
-class MultiSortingComparison(BaseComparison):
+class MultiSortingComparison(BaseMultiComparison, BaseSpikeTrainComparison):
```

Comment:
I don't think it is a good idea to add a cyclic design in hineritance, could have a discussion on that.